Description: classpath and parent project config
 * configuration from parent project
 * maven-jar-plugin adds Class-Path to MANIFEST.MF
Author: Eugene Zhukov <jevgeni.zh@gmail.com>
Last-Update: <2013-05-21>

--- a/pom.xml
+++ b/pom.xml
@@ -31,6 +31,10 @@
     </site>
   </distributionManagement>
 
+  <properties>
+    <configs.path>target/config</configs.path>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
@@ -56,6 +60,12 @@
   </dependencies>
 
   <build>
+    <resources>
+      <resource>
+        <directory>${configs.path}</directory>
+        <targetPath>META-INF</targetPath>
+      </resource>
+    </resources>
     <pluginManagement>
       <plugins>
         <plugin>
@@ -82,6 +92,29 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-component-metadata</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate-metadata</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+            <addClasspath>true</addClasspath>
+            </manifest>
+          </archive>
+        </configuration>
+        <version>2.2</version>
+      </plugin>
     </plugins>
   </build>
   
